### DISCUSSION TITLE
[<libxmlpp>] libxml3 support

### DISCUSSION THREAD

**Discussion by Baklap4** [microsoft/vcpkg#19402] [Q&A] [answered]
****

**Is your feature request related to a problem? Please describe.**
Are there any plans in supporting libxml++3?

**Proposed solution**
Build it when wanted

**Comment by PhoebeHui** [comment]
@Baklap4, vcpkg has update libxmlpp to lastest version https://download.gnome.org/sources/libxml++/5.0/, does that match your requirements?

**Comment by Baklap4** [comment]
Sadly i can't quite port to libxmlpp 5 as of yet :( I've tried to add it yesterday, but got into a dependency hell with glibmm
If it makes things easier: 3.2.3 has meson support

**Comment by PhoebeHui** [comment, marked as answer]
@Baklap4, we recommend to use [overlay ports](https://github.com/microsoft/vcpkg/blob/master/docs/specifications/ports-overlay.md#3-proposed-solution) instead, you can change the libxmlpp to the version you want in your own ports.

**Reply by Baklap4** [reply]
In the end we've ported our program to support and be able to link to libxmlpp 5.0 (was fairly easy) :)  thanks for the link to the docs!
